crates/sui-graphql-rpc/src/types/checkpoint.rs

```diff
@@ -117,8 +117,9 @@ impl Checkpoint {
     }
 
     /// The total number of transaction blocks in the network by the end of this checkpoint.
-    async fn network_total_transactions(&self) -> Option<UInt53> {
-        Some(self.network_total_transactions_impl().into())
+    async fn network_total_transactions(&self) -> Result<Option<UInt53>> {
```

Comment:
@amnn changing the return type here, any wisdom? I suppose we will need to handle this everywhere when we remove NOT NULL contraints for those in fact NOT NULL columns.